src/features/script-load.js

```diff
@@ -46,3 +46,17 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
   });
 };
 
+if (typeof window !== 'undefined') {
+  window.addEventListener('load', loadScriptModules);
+}
+
+function loadScriptModules() {
+  window.removeEventListener('load', loadScriptModules);
```

Comment:
I think we can remove for a one-off - footprint is more important than a few byte GC!